gen/definitions/banner.yaml

```diff
@@ -0,0 +1,9 @@
+---
+name: Banner
+path: Cisco-IOS-XR-um-banner-cfg:/banners/banner[banner-type=%s]
+doc_category: System
+attributes:
+  - yang_name: banner-type
+    example: prompt-timeout
+  - yang_name: line
+    example: ", banner-text ,"
```

Comment:
For the login banner example then maybe something like `"Hello user!"`.

Reply:
change the line to "Hello user ! "